### ISSUE TITLE
Part of welcome message is printed twice in console

### ISSUE THREAD

**Issue by EricSL** [PowerShell/Win32-OpenSSH#1375]
Please answer the following

**"OpenSSH for Windows" version**
Regression in 7.9, bug not observed in 7.7
 
**Server OperatingSystem**
Debian GNU/Linux 9.4

**Client OperatingSystem**
Windows 10

**What is failing**

When we log into the server our server prints a welcome message that in our case is around 30 lines of text including things like the operating system, last login, brief terms of use for the server, and an ANSI colored logo. On 7.9 (but not 7.7), part or all of this initial message is printed twice. If it is partial it is the start of the message, up to a variable number of lines, and after that things only print once.

**Comment by soul4soul** [comment]
I believe I'm seeing the same issue.

I perform the same steps using putty 0.70.0 client and only see the welcome message once.

**Client**
Windows 10, version 1909 client running OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5

**Server**
Ubuntu 18.04.3 LTS running OpenSSH_7.6p1 Ubuntu-4ubuntu0.3, OpenSSL 1.0.2n  7 Dec 2017

```
PS H:\> ssh engineer@se204
engineer@se204's password:
Welcome to Ubuntu 18.04.3 LTS (GNU/Linux 5.0.0-37-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage


 * Canonical Livepatch is available for installation.
   - Reduce system reboots and improve kernel security. Activate at:
     https://ubuntu.com/livepatch

41 packages can be updated.
0 updates are security updates.

Your Hardware Enablement Stack (HWE) is supported until April 2023.
Last login: Tue Dec 17 07:48:06 2019 from 172.16.20.158
Welcome to Ubuntu 18.04.3 LTS (GNU/Linux 5.0.0-37-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage


 * Canonical Livepatch is available for installation.
   - Reduce system reboots and improve kernel security. Activate at:
     https://ubuntu.com/livepatch

41 packages can be updated.
0 updates are security updates.

Your Hardware Enablement Stack (HWE) is supported until April 2023.
Last login: Tue Dec 17 07:48:06 2019 from 172.16.20.158
```

**Comment by mgkuhn** [comment]
Does this issue still exist in 8.1? (released this week via KB5003173 and Windows Update on the Semi-Annual Channel)

**Comment by bagajjal** [comment]
@EricSL  - This is fixed in next Github release V8.6. Meanwhile you can test this using the private binary listed in https://github.com/PowerShell/Win32-OpenSSH/issues/1736

**Comment by bagajjal** [comment]
Duplicate of https://github.com/PowerShell/Win32-OpenSSH/issues/1736
